### ISSUE TITLE
[Nice to Have] Support OBD2 modules

### ISSUE THREAD

**Issue by hakuchi** [mapillary/mapillary_issues#2393]
Can we support OBD2 modules(WiFi and Bluetooth) so we can work better in tunnels and with bad weather.

**Comment by gyllen** [comment]
@jesolem Thoughts?

**Comment by jesolem** [comment]
I think it is worth investigating.

**Comment by filipca** [comment]
I don't think it is possible but it would be great not to take pictures when the screen wipers pass through the camera view.

**Comment by jesolem** [comment]
@filipca I think that would be very difficult ;) We could detect and post-filter.

**Comment by HolgerJeromin** [comment]
Perhaps the wiper state is available via odb2. Would be cool to react to it.

**Comment by kozuch** [comment]
For inspiration see this: https://wiki.openstreetmap.org/wiki/OpenStreetCam/OBD2

**Comment by Kaldea** [comment]
_mapillary_issues_ is no longer an active repository.

Let’s continue discussing features in our forum! We will import this issue as topic to the [Mapillary Forum](http://forum.mapillary.com) in the coming weeks. We’ll notify here once it’s been done. See you there!
